os/drivers/mipidsi/Make.defs

```diff
@@ -0,0 +1,30 @@
+##########################################################################
+#
+# Copyright 2024 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+############################################################################
+
+ifeq ($(CONFIG_MIPI_DSI),y)
+
+CSRCS += mipi_dsi_device.c mipi_dsi_host.c mipi_dsi_packet.c mipi_dsi_device_driver.c mipi_dsi_host_driver.c
+
+ifeq ($(CONFIG_MIPI_DSI_DRIVER),y)
+	CSRCS += mipi_dsi_device_driver.c mipi_dsi_host_driver.c
```

Comment:
Please check alignment

Reply:
I checked other Make.defs. Different modules have used different standards like tab, four spaces, two spaces.
I have removed the tab from here. Please let me know what is the correct way for Make.defs. As the wiki link for code standards isn't working.